django/tests/otto/test_cleanup.py

```diff
@@ -267,6 +267,57 @@ def test_delete_old_chats_task(client, all_apps_user):
     assert Chat.objects.count() == 1
 
 
+@pytest.mark.django_db
+def test_delete_unused_libraries_task(client, all_apps_user, basic_user):
+    """
+    Test the delete_unused_libraries task.
+    """
+    user = all_apps_user()
+    # Create a Library by posting to the library route
+    client.force_login(user)
+    start_time = timezone.now()
+    url = reverse("librarian:modal_create_library")
+    response = client.post(
+        url, {"name_en": "New Library", "is_public": False, "order": 1}
+    )
+    assert response.status_code == 200
+    # Check that the library was created
+    from librarian.views import get_editable_libraries
+
+    user_libraries = get_editable_libraries(user)
+    assert len(user_libraries) == 3
+    library = user_libraries[2]
+    library_id = library.id
+    assert library is not None
+    time.sleep(3)
+    # Access the library again to update the accessed_at time
+    Library.objects.filter(id=library.id)
+    # library.refresh_from_db()
+    library.accessed_at = start_time + timezone.timedelta(seconds=10)
```

Comment:
You need to confirm that the "accessed_at" property is actually updated from the scenarios that we described:
a) when a property is changed on a Library/DataSource/Document, DataSource created, Document created
b) when a library is used in a Q&A mode chat response

If not already implemented, this needs to be implemented in the various views (librarian modal views and chat Q&A response views).

These routes should be exercised in the pytest, to ensure that the accessed_at property is actually updated.
You will have to use https://pypi.org/project/pytest-freezegun/ to manipulate the time.